server/routes/referrals/referralsController.ts

```diff
@@ -39,15 +45,67 @@ export default class ReferralsController {
   }
 
   async createReferral(req: Request, res: Response): Promise<void> {
-    const referral = await this.interventionsService.createDraftReferral(res.locals.user.token)
+    let error: FormValidationError | null = null
+
+    const crn = req.body['service-user-crn']
+    if (crn === null || crn === '') {
+      error = {
+        errors: [
+          {
+            formFields: ['service-user-crn'],
+            errorSummaryLinkedField: 'service-user-crn',
+            message: errorMessages.startReferral.crnEmpty,
+          },
+        ],
+      }
+    } else {
+      try {
+        await this.communityApiService.getOffenderByCRN(crn)
```

Comment:
using community api for simplicity until george's probation offender search changes are in